spec/database_helpers.rb

```diff
@@ -0,0 +1,7 @@
+require 'pg'
+
+def persisted_data(id:, table:)
+  connection = PG.connect(dbname: 'Chitter_Manager_test')
+  result = connection.query("SELECT * FROM peeps WHERE id = #{id};")
```

Comment:
You could refer to the 2nd parameter(`table:`) in your SQL query here, instead of hardcoding `peeps` so this helper script can be used for other tables too.